prow/cmd/deck/templates.go

```diff
@@ -66,6 +67,7 @@ func prepareBaseTemplate(o options, ca jobs.ConfigAgent, t *template.Template) (
 		"sections":         getConcreteSectionFunction(o),
 		"mobileFriendly":   func() bool { return true },
 		"mobileUnfriendly": func() bool { return false },
+		"deckVersion":      func() string { return version.Version },
```

Comment:
is `prowVersion` a better name for this?

Reply:
I went for `deckVersion` because we do not strictly know that all of prow's components are actually running at the same commit, and deck is the component we know for sure about. I don't have very strong opinions on this, though.

Reply:
make sense. i was thinking about maybe we could have a place for all the versions, like k/dashboard’s about page.

but for now, how about we label this on the siderbar to avoid any misunderstandings?

Reply:
Seems pretty self-evident to me that Deck would be displaying Deck's version, but 🤷‍♂️